app/Http/Controllers/SigsController.php

```diff
@@ -255,11 +255,20 @@ public static function getAllJson(Request $request)
 
         // see if request has been cached
         if (Cache::has("sigs-${order}")) {
-            return response()->json(Cache::get("sigs-${order}"));
+           return response()->json(Cache::get("sigs-${order}"));
```

Comment:
My only comment has to do with caching. As it is right now, cache will ignore the new `active` flag. Change the key (`"sigs-${order}"`) to include the flag, too, e.g. `"sigs-${order}-${active}"` both at `Cache::get()` and `Cache::put()`

Reply:
Should be fixed now!